### ISSUE TITLE
Bug on Scroll

### ISSUE THREAD

**Issue by denmasgie** [kakawait/hugo-tranquilpeak-theme#394]
### Configuration

 - **Operating system with its version**:  Android 6/arm64
 - **Browser with its version**:  Mozilla 68.2.0
 - **Hugo version**:hugo v0.59.1 
 - **Tranquilpeak version**: v0.47
 - **Do you reproduce on https://tranquilpeak.kakawait.com demo?**: Yes
 

### Actual behavior
When I open the sidebar and scroll on it its always closed,Then what if there is a dropdown in the sidebar whether this is a new problem or fix this problem

### Expected behavior
What I expect when I scroll the sidebar is always in a stationary position, not closing to its original position

### Steps to reproduce the behavior

**Comment by kakawait** [comment]
@denmasgie Yes I know that issue :'( I have to find why. I already open an issue on upstream project but without more luck.
